frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-details-page.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+
+import { navFactory } from '@console/internal/components/utils';
+
+import { DetailsPage } from '@console/internal/components/factory';
+import { K8sResourceKindReference } from '@console/internal/module/k8s';
+
+import { TemplateModel } from '@console/internal/models';
+import { VMTemplateDetailsFirehose } from './vm-template-details';
+import { VMDisksFirehose } from '../vm-disks';
+
+export const VMTemplateDetailsPage = (props: VMTemplateDetailsPageProps) => {
+  /* TODO(mlibra): pages will be transferred one by one in follow-ups
+  const nicsPage = {
+    href: 'nics',
+    name: 'Network Interfaces',
+    component: VmNic,
+  };
+  */
+
+  const disksPage = {
+    href: 'disks',
+    name: 'Disks',
+    component: VMDisksFirehose,
+  };
+
+  const pages = [
+    navFactory.details(VMTemplateDetailsFirehose),
+    navFactory.editYaml(),
+    // nicsPage,
+    disksPage,
```

Comment:
+1 adding the disksPage. Didn't expect it to work out of the box :)

Reply:
:+1: Works now, I think I dested it using a broken build ...